sklearn/compose/_target.py

```diff
@@ -274,14 +290,13 @@ def fit(self, X, y, **fit_params):
         if y_trans.ndim == 2 and y_trans.shape[1] == 1:
             y_trans = y_trans.squeeze(axis=1)
 
-        if self.regressor is None:
-            from ..linear_model import LinearRegression
```

Comment:
Interesting. I would have think that we have a circular import but apparently this is not the case anymore.